api/server.spec.js

```diff
@@ -0,0 +1,140 @@
+const request = require('supertest');
+const server = require('./server.js');
+const db = require('../data/dbConfig.js');
+
```

Comment:
Great job, you went above and beyond here. Like that you have multiple expects on almost all of them!